### PR TITLE
Add Cloud shell terminal like Che editor.

### PR DESCRIPTION
**Pull request by AndrienkoAleksandr**
### What does this PR do?
Add Cloud shell terminal like Che editor.

It can be tested with the following Devfile:
```yaml
metadata:
  name: cloud-shell
components:
  - reference: https://gist.githubusercontent.com/sleshchenko/c1cd908303b12684ac21e616c4cee1e9/raw/7e1dc5d106024af49fb1009af0c18792d2b1e328/meta.yaml
    type: cheEditor
    alias: cloudShell
  - memoryLimit: 256Mi
    type: dockerimage
    alias: dev
    image: 'eclipse/che-remote-plugin-openshift-connector-0.0.17:7.2.0'
    env:
      - value: '\[\e[34m\]>\[\e[m\]\[\e[33m\]>\[\e[m\]'
        name: PS1
apiVersion: 1.0.0

```
### Referenced issue:
https://github.com/eclipse/che/issues/15434
https://github.com/eclipse/che/issues/15497

### Depends on:
- [x] https://github.com/eclipse/che-machine-exec/pull/63
- [x] https://github.com/eclipse/che-machine-exec/pull/64
- [x] https://github.com/eclipse/che-machine-exec/pull/77

### CODE REVIEW

File: v3/plugins/eclipse/cloud-shell/che-machine-exec-plugin/nightly/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+apiVersion: v2
```
Comment (amisevsk) [inline v3/plugins/eclipse/cloud-shell/che-machine-exec-plugin/nightly/meta.yaml]: The directory structure you're using is incompatible with the `publisher/name/version` plugin IDs we use (name would be `cloud-shell/che-machine-exec-plugin`); perhaps just `eclipse/cloud-shell/nightly` is sufficient?

Comment (amisevsk) [reply v3/plugins/eclipse/cloud-shell/che-machine-exec-plugin/nightly/meta.yaml]: Actually, to match the publisher/name/version in meta.yaml, this meta.yaml *has* to be at `v3/plugins/eclipse/cloud-shell/nigthly/meta.yaml` or CI will fail.

Comment (AndrienkoAleksandr) [reply v3/plugins/eclipse/cloud-shell/che-machine-exec-plugin/nightly/meta.yaml]: Thanks, fixed.

File: v3/plugins/eclipse/cloud-shell/nightly/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+publisher: eclipse
+name: cloud-shell
+version: nightly
+type: Che Editor
+displayName: Cloud Shell Editor
+title: Cloud Shell Editor
+description: Cloud Shell Editor provides ab ability to use terminal widget instead of text editor.
+icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
+repository: https://github.com/eclipse/che-machine-exec/
+firstPublicationDate: "2019-12-17"
```
Comment (tolusha) [inline v3/plugins/eclipse/cloud-shell/nightly/meta.yaml]: `2020-01-28` ?

File: v3/plugins/eclipse/cloud-shell/latest.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+nightly
```
Comment (tolusha) [inline v3/plugins/eclipse/cloud-shell/latest.txt]: @amisevsk 
Is that version correct? 
Is it better to set something like 0.0.1?

Comment (AndrienkoAleksandr) [reply v3/plugins/eclipse/cloud-shell/latest.txt]: I guess correct version will be applied after release.

Comment (amisevsk) [reply v3/plugins/eclipse/cloud-shell/latest.txt]: I think setting `latest=nightly` makes sense here, it's a pre-release development build. The latest meta.yaml will be generated from the `nightly` meta.yaml during the registry build.

**Comment by sleshchenko** [comment]
@amisevsk Sometimes it works fine for me
Screenshot_20200129_121758

Sometimes it just hangs and I don't get my terminal working... I'll register a separate issue for cloud shell

**Pull request merged**